buildpipeline/pipeline.json

```diff
@@ -136,6 +136,33 @@
           }
         }
       ]
+    },
+    {
+      "Name": "Publish finalized build",
+      "Parameters": {
+        "TreatWarningsAsErrors": "false"
+      },
+      "BuildParameters": {
+        "BuildConfiguration": "Release"
+      },
+      "Definitions": [
+        {
+          "Name": "Core-Setup-Publish",
+          "Parameters": {
+            "PB_DistroRid": "win7-x86",
```

Comment:
There's some usages of `x86` here and some `x64`.  I think they should be consistent in this build def.